### PR TITLE
Added some more clarity around HA connections and failover.

### PR DESCRIPTION
**Pull request by daravep**

### CODE REVIEW

File: docs/2.9.0/docs/canton/usermanual/connectivity.rst
Comment (rgugliel-da) [inline docs/2.9.0/docs/canton/usermanual/connectivity.rst:123]:
```suggestion
sequencer. On the write path, the participant will send requests to one of the available sequencers in a round-robin manner, circling
```

**Review by rgugliel-da — APPROVED** [review]

File: docs/2.9.0/docs/canton/usermanual/connectivity.rst
Comment (marciw-da) [inline docs/2.9.0/docs/canton/usermanual/connectivity.rst:124]:
```suggestion
On the read side, the participant will establish a subscription to one available sequencer to subscribe to the stream of
encrypted messages. If the sequencer fails, the participant will automatically attempt to resubscribe to the next available
sequencer. On the write path, the participant will send requests to all available sequencers in a round-robin manner, circling
through the available sequencers. The same connectivity behavior applies to the domain manager and the mediator.
```

Comment (marciw-da) [reply docs/2.9.0/docs/canton/usermanual/connectivity.rst:124]: If it's really important to say that the round-robin happens "in sequence," maybe say "in order" instead. The phrase "available sequencers in sequence" is just not very readable.

**Review by marciw-da — APPROVED** [review]
Approving with two suggestions

**Comment by garyverhaegen-da** [comment]
We have [started the 3.1 documentation folder "for real"](https://github.com/digital-asset/docs.daml.com/pull/675). This pull requests makes changes to 2.9.0; please consider whether this should be forward-ported to 3.1 before merging it.

File: docs/2.9.0/docs/canton/usermanual/connectivity.rst
Comment (daravep) [reply docs/2.9.0/docs/canton/usermanual/connectivity.rst:124]: fixed

**Comment by daravep** [comment]
@garyverhaegen-da can you check? it seems that any docs change needs your approval. is this really intended?

**Review by carrielaben-da — APPROVED** [review]

**Comment by carrielaben-da** [comment]
@daravep: @garyverhaegen-da and I can both approve docs PRs, we're currently setting up @cathyjung-da to do so as well.

**Comment by carrielaben-da** [comment]
Basically, you just lost Marci's previous approval when we lost Marci.

**Pull request merged**
